contributing-docs/03_contributors_quick_start.rst

```diff
@@ -141,13 +140,14 @@ Docker Compose
 
   docker-compose --version
 
-Pyenv and setting up virtual-env
---------------------------------
-    Note: You might have issues with pyenv if you have a Mac with an M1 chip. Consider using virtualenv as an alternative.
+Setting up virtual-env
+----------------------
 
-1. Install pyenv and configure your shell's environment for Pyenv as suggested in Pyenv `README <https://github.com/pyenv/pyenv/blob/master/README.md#installation>`_
+1. While you can use any virtualenv manager, we recommend using `Hatch <https://hatch.pypa.io/latest/>`__
+   as your development environment front-end, and we already use Hatch backend ``hatchling`` for Airflow.
```

Comment:
nit: as your frontend development environment

Reply:
Well. It's a bit different and the naming is a bit tricky. Technically the terms that should be used are "build and integration frontend, and "build backend" following PEP-517 terminology. it is a bit confusing for some people as this discussion shows https://discuss.python.org/t/pep517s-definition-of-frontend-and-backend-is-unclear-to-me/16575/22. I think the conclusion from the discussion is that there should be some future change to explain it better and maybe change the terminology, I think it's better to stick to what PEP 517 states https://peps.python.org/pep-0517/#terminology-and-goals

I updated it and linked to the terminology from here.